src/Calculator/Views/GraphingCalculator/KeyGraphFeaturesPanel.xaml

```diff
@@ -306,28 +213,51 @@
 
         <converters:BooleanToVisibilityNegationConverter x:Name="BooleanToVisibilityNegationConverter"/>
     </UserControl.Resources>
-
-    <Grid Margin="0,7,0,0" Background="Transparent">
+    <Grid>
         <Grid.RowDefinitions>
             <RowDefinition Height="Auto"/>
             <RowDefinition Height="Auto"/>
             <RowDefinition Height="*"/>
         </Grid.RowDefinitions>
-        <controls:EquationTextBox x:Name="EquationInputTextBox"
-                                  x:Uid="EquationInputButton"
-                                  Grid.Row="0"
-                                  Margin="0,0,3,0"
-                                  Style="{StaticResource KGF_EquationTextBoxStyle}"
-                                  Background="Transparent"
-                                  BorderThickness="0"
-                                  DataContext="{x:Bind ViewModel, Mode=OneWay}"
-                                  EquationButtonClicked="EquationButtonClicked"
-                                  EquationButtonContentIndex="{x:Bind ViewModel.FunctionLabelIndex, Mode=OneWay}"
-                                  EquationColor="{x:Bind ViewModel.LineColor, Mode=OneWay}"
-                                  Loaded="EquationInputTextBox_Loaded"/>
+        <Grid Padding="0,4,0,0">
+            <Grid.ColumnDefinitions>
+                <ColumnDefinition Width="Auto"/>
+                <ColumnDefinition Width="*"/>
+            </Grid.ColumnDefinitions>
+            <Button x:Name="BackButton"
+                    x:Uid="equationAnalysisBack"
+                    MinWidth="44"
+                    MinHeight="44"
+                    VerticalAlignment="Stretch"
+                    Style="{StaticResource ButtonRevealStyle}"
+                    Background="{x:Bind ViewModel.LineColor}"
```

Comment:
In high contrast mode this should be the selected text background color instead of line color

Reply:
this is part of a different PR (Marco' s)